flask_static_digest/__init__.py

```diff
@@ -55,10 +54,19 @@ def _load_manifest(self, endpoint, scaffold):
         except (FileNotFoundError, Exception):
             pass
 
-    def _prepend_host_url(self, host, filename):
-        return urljoin(
-            self.host_url, "/".join([self.static_url_path, filename])
-        )
+    def _custom_url_join(self, url):
```

Comment:
Can you please rename the `url` argument to `url_path`?

Reply:
This contradicts your suggestion in https://github.com/nickjj/flask-static-digest/pull/38#discussion_r1604143170. Should I rename it to `url_for` or `url_path`?